### ISSUE TITLE
expose collection queryables

### ISSUE THREAD

**Issue by jdries** [Open-EO/openeo-geopyspark-driver#536]
Users get warnings because properties are not correctly listed in STAC summaries:
`UserWarning: SENTINEL1_GRD property filtering with properties that are undefined in the collection metadata (summaries): relativeOrbitNumber.`

It's also problematic that users don't know what properties can be filtered on.

**Comment by soxofaan** [comment]
FYI: openEO API 1.2 adds:

> New endpoint for metadata filters (queryables): /collections/{collection_id}/queryables. Also adds a new rel type to the collection links. https://github.com/Open-EO/openeo-api/issues/396

**Comment by soxofaan** [comment]
I added it to the backlog of our planning system to put it higher on our radar.

What kind of properties do you see most in use practically?

**Comment by clausmichele** [comment]
cloud cover and tile id for Sentinel-2. Orbit for Sentinel-1. But in general, it would be really important to document the available ones somewhere, maybe in the collection metadata so that they are also rendered with the other info in the openEO clients.

**Comment by soxofaan** [comment]
I fully agree that this is a useful addition. It's just a considerable amount of work to get all that metadata in place.
(that's why I'm fishing for some low hanging fruit like cloud cover, which would be easier to get in place initially, compared to a more general solution)
